src/cryptoadvance/specter/server_endpoints/settings.py

```diff
@@ -387,6 +397,15 @@ def tor():
     )
 
 
+@settings_endpoint.route("/update_tor_type", methods=["POST"])
+@login_required
+def update_tor_type():
+    tor_type = request.form["torType"]
+    app.specter.update_tor_type(tor_type, current_user)
+    response = {"torTypeChanged": True, "error": None}
```

Comment:
I don't like `"error": None`. Did we introduced that as a pattern at some time? I hope not. Would prefer to simply not have that key in case on not having an error.

Reply:
It is not needed, will remove it.